canvases/dark-editor/index.dark

```diff
@@ -61,19 +66,144 @@
         background-color: #f2f2f2;
         color: #aaa;
       }
+      form{
+        height:80%;
+      }
     </style>
   </head>
   <body>
     <div class="split-container">
       <div class="left-half">
         <form id="gpt4-form">
           <input type="text" name="dark-editor" value="dark-editor" />
+          <textarea id="dark-prompt" name="dark-prompt">
```

Comment:
Maybe it is better to figure out a way to read the prompt from a file rather than pasting it here

Reply:
Yeah, in a separate PR you could:
- add an http GET handler to fetch the prompt on disk
- add some JS in the front-end part of this that fetches that prompt, and inserts it there

Reply:
That worked well, thanks!